Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs

```diff
@@ -271,558 +145,490 @@ public IEnumerable<UserAc> GetAllUsers()
                 user.NumberOfCasualLeave = editedUser.NumberOfCasualLeave;
                 user.NumberOfSickLeave = editedUser.NumberOfSickLeave;
                 user.SlackUserName = editedUser.SlackUserName;
-                var userPreviousInfo = await _userManager.FindByEmailAsync(editedUser.Email);
                 await _userManager.UpdateAsync(user);
+                //get user roles
                 IList<string> listofUserRole = await _userManager.GetRolesAsync(user);
-                var removeFromRole = await _userManager.RemoveFromRoleAsync(user, listofUserRole.FirstOrDefault());
+                //remove user role 
+                var removeFromRole = await _userManager.RemoveFromRoleAsync(user, listofUserRole.First());
+                //add new role of user.
                 var addNewRole = await _userManager.AddToRoleAsync(user, editedUser.RoleName);
```

Comment:
This method belong to someone else but still variable was not in use. I have removed it